src/mlpack/tests/recurrent_network_test.cpp

```diff
@@ -572,8 +583,8 @@ void DistractedSequenceRecallTestNetwork()
   {
     RNN<MeanSquaredError<> > model(rho);
     model.Add<IdentityLayer<> >();
-    model.Add<Linear<> >(inputSize, 14);
-    model.Add<LSTM<> >(14, 7, rho);
+    model.Add<Linear<> >(inputSize, 16);
```

Comment:
I tried some tests... to my surprise Reber grammar tests pass with just 1 unit of each. Distracted sequence tests pass with 3 units.